.github/CODEOWNERS

```
@@ -1,4 +1 @@
-# CODEOWNERS: https://help.github.com/articles/about-codeowners/
-
-* @chadoh
-* @vgrichina
+.github/workflows/ @morgsmccauley @andy-haynes
```

```suggestion
@morgsmccauley @andy-haynes
```

Should this not be there?

---

This is intentional - I only want this group to be notified when changes are made to `.github/workflows` (our Github Actions)